source/features/mark-private-repos.css

```diff
@@ -0,0 +1,4 @@
+li.private .Label--secondary {
+	border-color: var(--color-attention-muted);
+	color: var(--color-attention-emphasis);
```

Comment:
Looks good to me, but I'd want you to make sure that these are the colors GitHub uses in this case, I know that there's a specific "border" color type as well.

Reply:
😉 
<img width="331" alt="image" src="https://github.com/refined-github/refined-github/assets/50487467/a75b3734-df02-4c3e-af2c-df947ef304e3">

Reply:
And the `Attention Label` is used for `Public archive` repositories.
Is Private repository label colors can be same with these?

Reply:
FYI, other colors 
<img width="359" alt="image" src="https://github.com/refined-github/refined-github/assets/50487467/b7179111-c222-4ff8-804e-fc24d9cf3c16">